### PR TITLE
Vim: Jump post

### PR DESCRIPTION
**Pull request by duggiefresh**

### CODE REVIEW

File: source/posts/2015-01-31-vim-jump.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+open or previously visited files.
+
+Let's briefly cover the some of the most familiar ones.
+
+## File Jumps
+
+*File jumps* will navigate you to a location within the current file, regardless if that
+location is seen or not seen within the window.
+
+### <a href="https://www.youtube.com/watch?v=Xz-UvQYAmbg" target="_blank">Ain't no Mountain high enough, ain't no valley low enough...</a>
```
Comment (rsocci) [inline source/posts/2015-01-31-vim-jump.md]: nice

**Comment by bcardarella** [comment]
:+1: looks good to me, I can't really comment on some of the vim stuff as I'm not a power user like Doug

File: source/posts/2015-01-31-vim-jump.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+# <a href="https://www.youtube.com/watch?v=010KyIQjkTk" target="_blank">Jump, Jump, Jump!</a>
+
+In the last Vim-related post, we
+[discussed **mark** motion](http://reefpoints.dockyard.com/2014/04/10/vim-on-your-mark.html),
+and today, we're going to cover another type of navigation: **jump** motion.
+
+The main benefit of jump motion is its speed; it allows us to quickly traverse through the current file
+open or previously visited files.
+
+Let's briefly cover the some of the most familiar ones.
```
Comment (brzpegasus) [inline source/posts/2015-01-31-vim-jump.md]: Extra "the"

File: source/posts/2015-01-31-vim-jump.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+**/**
+
+* Allows you to search *forwards* for a desired pattern within the file.
+  * **/fishsticks** - Searches for all occurences of `fishsticks` ahead of your current cursor.
+
+**?**
+
+* Allows you to search *backwards* for a desired pattern within the file.
+  * **?catdog** - Searches for all occurences of `catdog` behind of your current cursor.
```
Comment (michaeldupuisjr) [inline source/posts/2015-01-31-vim-jump.md]: behind your current

File: source/posts/2015-01-31-vim-jump.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+* Jumps your cursor to the **middle** line of the window.
+
+**L**
+
+* Jumps your cursor to the **lowest** line of the window.
+
+## System Wide Jumps
+
+*System* jumps are special; they have the ability to take us any previously visited file,
```
Comment (michaeldupuisjr) [inline source/posts/2015-01-31-vim-jump.md]: take us *to* any

File: source/posts/2015-01-31-vim-jump.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+regardless if those files are or are not within the same directory.
+
+This where jump motion really shines!
```
Comment (michaeldupuisjr) [inline source/posts/2015-01-31-vim-jump.md]: This *is* where

File: source/posts/2015-01-31-vim-jump.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+![](https://i.imgur.com/mFc1cHz.png)
+
+Above is an example of a jump list. There are four columns: *jump*, *line*, *col* and *file/text*.
+The numbers underneath the *jump* column are used to prefix our jump command, **CTRL-O** and **CTRL-L**.
+We are also given the position of our cursor from the  *line* and *col*umn columns. Lastly, the
+*file/text* column, gives us either the file path or, if the jump is located in our currently opened file,
+the line of text.
+
+Using our example jump list, if we want to jump to `4`th jump, located within `~/dir2/file.md`, we'd
```
Comment (michaeldupuisjr) [inline source/posts/2015-01-31-vim-jump.md]: to *the* `4`th

**Comment by michaeldupuisjr** [comment]
The people are going to be vimming so hard when this post goes live. Nice work @duggiefresh!

**Comment by duggiefresh** [comment]
Thanks for the fixes @michaeldupuisjr! You have some great :eyes:!

**Comment by danmcclain** [comment]
@duggiefresh Can you update the date for today?

**Comment by duggiefresh** [comment]
Done @danmcclain :ship:

**Pull request merged**